README.md

```diff
@@ -20,3 +20,19 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+## Answer 
+Let's start with the best-case runtime analysis. For the best case that would be a sorted list that was put through the algorithm. This will give you a run time $O(n)$. 
```

Comment:
This still doesn't look right.

Reply:
I think I got it right I just had to look at my code and think about if I were to put an array through it physically.